src/pages/settings/InitialSettingsPage.tsx

```diff
@@ -102,6 +103,7 @@ function InitialSettingsPage({currentUserPersonalDetails}: InitialSettingsPagePr
 
     useEffect(() => {
         Wallet.openInitialSettingsPage();
+        App.confirmReadyToOpenApp();
```

Comment:
This is for when you open the App directly in the main settings page:

<img width="1581" alt="image" src="https://github.com/user-attachments/assets/81b51d5c-1896-4c32-ace4-35310b8aaf8f">